docker-compose.yaml

```diff
@@ -0,0 +1,130 @@
+version: "3.8"
+services:
+    ollama-server:
+        image: ollama/ollama:0.1.39
+        pull_policy: always
+        restart: always
+        container_name: ollama
+        ports:
+            - "${OLLAMA_PORT}:11434"
+        volumes:
+            - ./volumes/ollama:/root/.ollama
+        healthcheck:
+            test: ["CMD", "curl", "-f", "http://localhost:${OLLAMA_PORT}"]
+            interval: 1m
+            timeout: 10s
+            retries: 3
+        environment:
+            - OLLAMA_KEEP_ALIVE=${OLLAMA_KEEP_ALIVE}
+            - OLLAMA_HOST=${OLLAMA_HOST}
+        entrypoint: /bin/bash
+        command: -c "set -e; ollama serve & sleep 10 && ollama pull ${OLLAMA_MODEL} && sleep infinity"
+        networks:
+            - cognita-docker
+
+    infinity-server:
+        image: michaelf34/infinity:0.0.36
+        pull_policy: always
+        restart: always
+        container_name: infinity
+        ports:
+            - "${INFINITY_PORT}:7997"
+        volumes:
+            - ./volumes/infinity:/app/.cache
+        healthcheck:
+            test:
+                [
+                    "CMD",
+                    "curl",
+                    "-f",
+                    "http://localhost:${INFINITY_PORT}/health",
```

Comment:
Afaik healthchecks are executed inside the container so this should be port inside the container on which infinity is listening to
same for other healthchecks

Reply:
done.